packages/orbit-components/src/TextLink/TextLink.stories.tsx

```diff
@@ -1,292 +1,202 @@
-  href: "https://kiwi.com",
-  children: "TextLink with icon",
-  icon: "ChevronForward",
-};
+  args: {
+    iconRight: "ChevronForward",
+  },
 
-LinkWithRightIcon.argTypes = {
-  icon: {
-    options: Object.keys(Icons),
-    control: {
-      type: "select",
+  argTypes: {
+    iconRight: {
+      ...LinkWithLeftIcon.argTypes?.iconLeft,
     },
   },
 };
 
-export const Playground = ({
-  href,
-  type,
-  size,
-  external,
-  children,
-  rel,
-  iconRight,
-  iconLeft,
-  dataTest,
-  tabIndex,
-  stopPropagation,
-  standAlone,
-  noUnderline,
-}) => {
-  const IconRight = getIcon(iconRight);
-  const IconLeft = getIcon(iconLeft);
-  return (
-    <Box
-      padding="300"
-      borderRadius="100"
-      background={type === TYPE_OPTIONS.WHITE ? "inkLight" : "cloudNormal"}
-    >
-      <TextLink
-        external={external}
-        onClick={action("clicked")}
-        href={href}
-        type={type}
-        size={size}
-        rel={validate(rel)}
-        iconRight={IconRight && <IconRight />}
-        iconLeft={IconLeft && <IconLeft />}
-        dataTest={dataTest}
-        tabIndex={tabIndex}
-        stopPropagation={stopPropagation}
-        standAlone={standAlone}
-        noUnderline={noUnderline}
-      >
-        {children}
-      </TextLink>
-    </Box>
-  );
-};
-
-Playground.story = {
-  parameters: {
-    info: "You can try all possible configurations of this component. However, check Orbit.Kiwi for more detailed design guidelines.",
+export const TextLinkInText: Story = {
+  render: ({ children, ...args }) => {
+    return (
+      <Text>
+        Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Nullam dapibus fermentum ipsum.
+        Duis ante orci, molestie vitae vehicula venenatis, tincidunt ac pede.{" "}
+        <TextLink {...args}>{children}</TextLink>, fermentum vitae, sagittis id, malesuada in, quam.
+        Vivamus luctus egestas leo. Integer imperdiet lectus quis justo. Duis condimentum augue id
+        magna semper rutrum. Quisque porta. Sed elit dui, pellentesque a, faucibus vel, interdum
+        nec, diam.
+      </Text>
+    );
   },
-};
-
-Playground.args = {
-  href: "https://kiwi.com",
-  type: TYPE_OPTIONS.SECONDARY,
-  size: SIZE_OPTIONS.SMALL,
-  external: true,
-  children: "Custom link",
-  rel: "",
-  iconRight: "ChevronForward",
-  iconLeft: "",
-  dataTest: "test",
-  tabIndex: "",
-  stopPropagation: false,
-  standAlone: false,
-  noUnderline: false,
-};
 
-Playground.argTypes = {
-  type: {
-    options: Object.values(TYPE_OPTIONS),
-    control: {
-      type: "select",
+  parameters: {
+    info: "An example of usage of TextLink in Text component. Check Orbit.Kiwi for more detailed guidelines.",
+    controls: {
+      exclude: ["onClick", "external", "asComponent", "stopPropagation"],
     },
   },
-  size: {
-    options: Object.values(SIZE_OPTIONS),
-    control: {
-      type: "select",
-    },
+
+  args: {
+    type: TYPE_OPTIONS.SUCCESS,
+    size: SIZE_OPTIONS.SMALL,
```

Comment:
Small nitpick, I would set this to `NORMAL` so it's not smaller than the rest of the text.